ts/src/pro/bitget.ts

```diff
@@ -705,6 +706,182 @@ export default class bitget extends bitgetRest {
         }, market);
     }
 
+    async watchPositions (symbols: string[] = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name bitget#watchPositions
+         * @description watch all open positions
+         * @see https://bitgetlimited.github.io/apidoc/en/mix/#positions-channel
+         * @param {[string]|undefined} symbols list of unified market symbols
+         * @param {object} params extra parameters specific to the bitget api endpoint
+         * @param {string} params.instType Instrument Type umcbl:USDT Perpetual Contract Private Channel; dmcbl:Coin Margin Perpetual Contract Private Channel; cmcbl: USDC margin Perpetual Contract Private Channel
+         * @returns {[object]} a list of [position structure]{@link https://docs.ccxt.com/en/latest/manual.html#position-structure}
+         */
+        await this.loadMarkets ();
+        let market = undefined;
+        let messageHash = '';
+        const subscriptionHash = 'positions';
+        let instType = 'dmcbl';
+        if (!this.isEmpty (symbols)) {
```

Comment:
@pcriadoperez same thing here, and it even returns an empty array 
![image](https://github.com/ccxt/ccxt/assets/43336371/16f09744-c54a-4f19-a5c2-e1fc9466dbd8)

Reply:
```python
bitget watchPositions
[{'collateral': None,
  'contractSize': 1.0,
  'contracts': 1.8,
  'datetime': '2023-09-25T00:25:37.747Z',
  'entryPrice': 63.52,
  'hedged': True,
  'id': '926036334386778112',
  'info': {'achievedProfits': '0',
           'autoMargin': 'off',
           'available': '1.8',
           'averageOpenPrice': '63.52',
           'cTime': '1656510187717',
           'holdMode': 'double_hold',
           'holdSide': 'long',
           'instId': 'LTCUSDT_UMCBL',
           'instName': 'LTCUSDT',
           'keepMarginRate': '0.010',
           'leverage': 2,
           'liqPx': '1.76',
           'locked': '0',
           'margin': '57.168',
           'marginCoin': 'USDT',
           'marginMode': 'crossed',
           'marginRate': '0.024980468411',
           'markPrice': '63.29',
           'posId': '926036334386778112',
           'total': '1.8',
           'uTime': '1695601537747',
           'upl': '-0.414',
           'uplRate': '-0.0072'},
  'initialMargin': None,
  'initialMarginPercentage': None,
  'leverage': 2.0,
  'liquidationPrice': None,
  'maintenanceMargin': None,
  'maintenanceMarginPercentage': 0.01,
  'marginMode': 'cross',
  'marginRatio': 0.024980468411,
  'markPrice': 63.29,
  'notional': None,
  'percentage': -0.0072,
  'side': 'long',
  'symbol': 'LTC/USDT:USDT',
  'timestamp': 1695601537747,
  'unrealizedPnl': -0.414},
 {'collateral': None,
  'contractSize': 1.0,
  'contracts': 1.5,
  'datetime': '2023-09-25T00:18:14.998Z',
  'entryPrice': 63.82,
  'hedged': True,
  'id': '926036334386778114',
  'info': {'achievedProfits': '0',
           'autoMargin': 'off',
           'available': '1.5',
           'averageOpenPrice': '63.82',
           'cTime': '1656510187717',
           'holdMode': 'double_hold',
           'holdSide': 'short',
           'instId': 'LTCUSDT_UMCBL',
           'instName': 'LTCUSDT',
           'keepMarginRate': '0.010',
           'leverage': 2,
           'liqPx': '1.76',
           'locked': '0',
           'margin': '47.8695',
           'marginCoin': 'USDT',
           'marginMode': 'crossed',
           'marginRate': '0.024980468411',
           'markPrice': '63.29',
           'posId': '926036334386778114',
           'total': '1.5',
           'uTime': '1695601094998',
           'upl': '0.8039',
           'uplRate': '0.0167'},
  'initialMargin': None,
  'initialMarginPercentage': None,
  'leverage': 2.0,
  'liquidationPrice': None,
  'maintenanceMargin': None,
  'maintenanceMarginPercentage': 0.01,
  'marginMode': 'cross',
  'marginRatio': 0.024980468411,
  'markPrice': 63.29,
  'notional': None,
  'percentage': 0.0167,
  'side': 'short',
  'symbol': 'LTC/USDT:USDT',
  'timestamp': 1695601094998,
  'unrealizedPnl': 0.8039},
 {'collateral': None,
  'contractSize': 1.0,
  'contracts': 0.05,
  'datetime': '2023-09-25T00:46:57.080Z',
  'entryPrice': 1623.58,
  'hedged': True,
  'id': '1080971971354304526',
  'info': {'achievedProfits': '0',
           'autoMargin': 'off',
           'available': '0.05',
           'averageOpenPrice': '1623.58',
           'cTime': '1693449722135',
           'holdMode': 'double_hold',
           'holdSide': 'short',
           'instId': 'ETHUSDT_UMCBL',
           'instName': 'ETHUSDT',
           'keepMarginRate': '0.005',
           'leverage': 20,
           'liqPx': '3101.55',
           'locked': '0',
           'margin': '4.0589',
           'marginCoin': 'USDT',
           'marginMode': 'crossed',
           'marginRate': '0.024980468411',
           'markPrice': '1572.74',
           'posId': '1080971971354304526',
           'total': '0.05',
           'uTime': '1695602817080',
           'upl': '2.5424',
           'uplRate': '0.6263'},
  'initialMargin': None,
  'initialMarginPercentage': None,
  'leverage': 20.0,
  'liquidationPrice': None,
  'maintenanceMargin': None,
  'maintenanceMarginPercentage': 0.005,
  'marginMode': 'cross',
  'marginRatio': 0.024980468411,
  'markPrice': 1572.74,
  'notional': None,
  'percentage': 0.6263,
  'side': 'short',
  'symbol': 'ETH/USDT:USDT',
  'timestamp': 1695602817080,
  'unrealizedPnl': 2.5424},
 {'collateral': None,
  'contractSize': 1.0,
  'contracts': 0.02,
  'datetime': '2023-08-31T05:17:30.488Z',
  'entryPrice': 1700.62,
  'hedged': True,
  'id': '1080971971354304528',
  'info': {'achievedProfits': '0',
           'autoMargin': 'off',
           'available': '0.02',
           'averageOpenPrice': '1700.62',
           'cTime': '1693449722135',
           'holdMode': 'double_hold',
           'holdSide': 'long',
           'instId': 'ETHUSDT_UMCBL',
           'instName': 'ETHUSDT',
           'keepMarginRate': '0.005',
           'leverage': 20,
           'liqPx': '3101.55',
           'locked': '0',
           'margin': '1.7006',
           'marginCoin': 'USDT',
           'marginMode': 'crossed',
           'marginRate': '0.024980468411',
           'markPrice': '1572.74',
           'posId': '1080971971354304528',
           'total': '0.02',
           'uTime': '1693459050488',
           'upl': '-2.5576',
           'uplRate': '-1.5039'},
  'initialMargin': None,
  'initialMarginPercentage': None,
  'leverage': 20.0,
  'liquidationPrice': None,
  'maintenanceMargin': None,
  'maintenanceMarginPercentage': 0.005,
  'marginMode': 'cross',
  'marginRatio': 0.024980468411,
  'markPrice': 1572.74,
  'notional': None,
  'percentage': -1.5039,
  'side': 'long',
  'symbol': 'ETH/USDT:USDT',
  'timestamp': 1693459050488,
  'unrealizedPnl': -2.5576}]
  ```